Doc/library/enum.rst

```diff
@@ -553,7 +553,7 @@ Data Types
       of two, starting with ``1``.
 
    .. versionchanged:: 3.11  The *repr()* of zero-valued flags has changed.  It
-      is now::
+      is now:
```

Comment:
This should be two `:`s to trigger the proper formatting for the following block (like in line 601 below).

Reply:
That is what I thought, but two `:` appear (`::`) in the rendered HTML documentation: https://docs.python.org/3/library/enum.html?highlight=enum#enum.Flag._numeric_repr_

That is why I removed one `:`.

Reply:
Now I understand where the rendering problem (`::` was not replaced by `:` in the HTML output) came from: the following literal block was 4-space indented instead of 3-space indented. Fixed.